projects/transpose matrix/index.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Transpose matrix in Every Language
+layout: default
+date: 2020-10-03
+last-modified: 2020-10-03
+featured-image:
+tags: [transposematrix]
```

Comment:
This is personal preference, but I think we should use this style for consistency. 

```suggestion
tags: [transpose-matrix]
```